src/components/common/Table/ColumnHidePage/ColumnHidePage.tsx

```diff
@@ -0,0 +1,103 @@
+import { Checkbox, createStyles, FormControlLabel, IconButton, makeStyles, Popover, Theme, Tooltip, Typography } from "@material-ui/core";
+import { FunctionComponent, useState } from "react";
+import { TableInstance } from "react-table";
+import ViewColumnIcon from '@material-ui/icons/ViewColumn';
+
+const ID_COLUMNS_HIDE = 'ArtHan/ID_COLUMNS_HIDE';
+
+const useStyle = makeStyles((theme: Theme) => {
+  return createStyles({
+    columns_pop_over: {
+      padding: 15,
+    },
+    popover_title: {
+      fontSize: 20,
+      fontWeight: 700,
+      textTransform: 'uppercase',
+      textDecoration: 'underline',
+      marginBottom: 15
+    },
+    grid: {
+      display: 'grid',
+      gridTemplateColumns: 'repeat(3, 255px)',
+      gridColumnGap: 15,
+      gridRowGap: 15,
+    },
+  })
+})
+
+type ColumnHidePageProps = {
+  instance: TableInstance
+}
+
+const ColumnHidePage: FunctionComponent<ColumnHidePageProps> = ({ instance }) => {
+  const { allColumns, toggleHideColumn } = instance;
+
+  const classes = useStyle({});
+
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+  const [columnsOpen, setColumnsOpen] = useState(false);  
+
+  const handleColumnsClick = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
add an import statement (import React from 'react') || (import { MouseEvent } from 'react' and delete word React)